# Number of the finger, refer to the manipulator's specific layout
uint8 finger_number

# Fluid pressure [Pa]
float64 fluid_pressure

# Dynamic pressure [Pa]
float64 dynamic_pressure
